### PR TITLE
Update kotlin reference + Logger Swift reference

### PR DESCRIPTION
**Pull request by jfilowk**
Complete some examples for Kotlin Reference and update some old reference

Update Logger Swift reference with the last changes
https://github.com/mobilejazz/harmony-swift/pull/18

### CODE REVIEW

**Comment by jfilowk** [comment]
@joselufo Updated Logger documentation with all platforms

File: pages/Logger.md [hunk 1]
```diff
@@ -45,27 +97,144 @@ enum LogLevel {
...
+```typescript
 // TypeScript
+
 interface Logger {
     logKeyValue(key: string, value: any): void;
 
     log(level: LogLevel, message: string): void;
     log(level: LogLevel, tag: string, message: string): void;
 }
-```
```
Comment (doup) [inline pages/Logger.md]: We should keep this closing code block fence (```) here

File: pages/Logger.md [hunk 2]
```diff
@@ -45,27 +97,144 @@ enum LogLevel {
...
 
 - `logKeyValue`: logs a key-value pair.
 - `log`: method has two signatures, the difference being an optional `tag` parameter.
+```
```
Comment (doup) [inline pages/Logger.md]: And remove this one.

File: pages/Logger.md [hunk 3]
```diff
@@ -45,27 +97,144 @@ enum LogLevel {
...
+    }
+}
+```
+
+### Alias methods (Typescript)
+
+```typescript
 // TypeScript
+
+// There are also few alias methods available, one for each log level, all of them with an optional first parameter: `tag`.
```
Comment (doup) [inline pages/Logger.md]: This should go outside the code block.

Comment (doup) [reply pages/Logger.md]: Move above, as it was before.

File: pages/Logger.md (hunk 1 above)
Comment (jfilowk) [reply pages/Logger.md]: done

File: pages/Logger.md (hunk 3 above)
Comment (jfilowk) [reply pages/Logger.md]: moved

File: pages/Logger.md (hunk 2 above)
Comment (jfilowk) [reply pages/Logger.md]: done

**Pull request merged**